src/main/scala/com.gu.gibbons/services/interpreters/BonoboInterpreter.scala

```diff
@@ -30,7 +30,7 @@ class BonoboInterpreter(config: Settings,
 
   def getUsers(jadis: Instant) =
```

Comment:
I don't know what `jadis` means either.
 If anyone wants to change that to something which describes what it is, then please do.

Reply:
I very naively thought it was some sort of Scala terminology but apparently it's just French for "long ago?" :D. I renamed it everywhere in the codebase to more descriptive names.

Reply:
`jadis` means `before long time ago`, you can translate it in this case to `before`